server/chat-commands/admin.ts

```diff
@@ -299,9 +299,9 @@ export const commands: ChatCommands = {
 				// rebuild the formats list
 				delete Rooms.global.formatList;
 				// respawn validator processes
-				void TeamValidatorAsync.PM.respawn();
+				await TeamValidatorAsync.PM.respawn();
 				// respawn simulator processes
-				void Rooms.PM.respawn();
+				await Rooms.PM.respawn();
 				// broadcast the new formats list to clients
```

Comment:
Should these run in parallel?

Reply:
I agree, these should probably be in parallel.

Reply:
`Promise.all` will let you parallelize them.